### PR TITLE
Test @include and @except

### PR DESCRIPTION
**Pull request by HelenaSabel**
This PR adds the test mentioned in issue #140 to the test suite. It includes:
- A slight modification to the ODD created by @lb42 : [Test/testClass.odd](https://github.com/TEIC/Stylesheets/blob/issue-140-except/Test/testClass.odd)
- The addition of the test to `Makefile` (with the required modification of the expected results).

### CODE REVIEW

File: Test/Makefile [hunk 1]
```diff
@@ -753,8 +753,16 @@ test-namespaces: actual
 	perl -p -i -e 's/generated from ODD source .*//' $(AR)/test35.rnc
 	perl -i -pe 'BEGIN{undef $$/;} s/# ?Schema[^#]+#[^#]+#[^#]+#[^#]+#\n//smg' $(AR)/test35.rnc
 	if [ $(DIFFNOW) -eq 1 ]; \
-		then diff -bBw $(AR)/test35.rnc $(ER)/test35.rnc; \
-		else echo "==deferring: \` diff -bBw $(AR)/test35.rnc $(ER)/test35.rnc \`"; fi
+	then diff -bBw $(AR)/test35.rnc $(ER)/test35.rnc; \
+	else echo "==deferring: \` diff -bBw $(AR)/test35.rnc $(ER)/test35.rnc \`"; fi
```
Comment (sydb) [inline Test/Makefile]: Indentation should probably be restored. 😊

File: Test/testClass.odd [hunk 2]
```diff
@@ -45,15 +45,25 @@
...
                <classes>
                   <memberOf key="model.divPart"/>
                </classes>
                <content>
                   <classRef key="testClass"/>
                </content>
             </elementSpec>
+            
+            <elementSpec ident="testP1b">
+               <desc>Alternate: should generate x | y | z</desc>
```
Comment (sydb) [inline Test/testClass.odd]: May as well match the attribute value, no? I.e. “Alternation:”.

File: Test/Makefile [hunk 3]
```diff
@@ -753,8 +753,16 @@ test-namespaces: actual
...
+	
+#HBS 2022-04-30 tests attributes @include and @except
+	$(BINDIR)/teitornc $(FLAGS) testClass.odd $(AR)/testClass.rnc
+	perl -p -i -e 's/generated from ODD source .*//' $(AR)/testClass.rnc
+	perl -i -pe 'BEGIN{undef $$/;} s/# ?Schema[^#]+#[^#]+#[^#]+#[^#]+#\n//smg' $(AR)/testClass.rnc
```
Comment (sydb) [inline Test/Makefile:762]: I realize this regexp fix was just copied from another file (test21, test15, test30 all use that same icky regex), and I do _not_ think we should do anything different here in this PR. But we should move this from here into cleanup.pl, (on a different issue), no?

**Review by sydb — APPROVED** [review]

**Comment by peterstadler** [comment]
Just a reminder to @HelenaSabel to react to @sydb's questions and proposals ;)

File: Test/Makefile (hunk 1 above)
Comment (HelenaSabel) [reply Test/Makefile]: Good catch! I don’t even how I changed the indentation of the lines preceding my addition...

File: Test/Makefile (hunk 3 above)
Comment (HelenaSabel) [reply Test/Makefile:762]: created issue #580

**Comment by HelenaSabel** [comment]
Thank you for the kind poke, @peterstadler. I think I addressed all of @sydb’s comments.

**Pull request merged**